src/utils/mavenUtils.ts

```diff
@@ -59,6 +60,13 @@ export async function pluginDescription(pluginId: string, pomPath: string): Prom
     return await readFileIfExists(outputPath);
 }
 
+export async function rawProfileList(pomPath: string): Promise<string | undefined> {
+    const outputPath: string = getTempFolder(pomPath);
+    const profileListPath = `${outputPath}.profiles.txt`;

```

Comment:
Is this a temp file to store the output? Will it be deleted after execution?

Reply:
Yes there is. But it won't be deleted for below reasons:
- In logs, you know what command was executed and which files are generated. You can open it to check the content if error occurs.
- For each project we generate at most one file, and it will be overwritten in future executions. So no much concern about storage.

Reply:
Ok